framework/src/variables/MooseVariableFV.C

```diff
@@ -438,16 +443,19 @@ MooseVariableFV<OutputType>::getDirichletBC(const FaceInfo & fi) const
       .queryInto(bcs);
   mooseAssert(bcs.size() <= 1, "cannot have multiple dirichlet BCs on the same boundary");
 
-  bool has_dirichlet_bc = bcs.size() > 0;
+  const bool has_dirichlet_bc = bcs.size() > 0;
 
   if (has_dirichlet_bc)
   {
     mooseAssert(bcs[0], "The FVDirichletBC is null!");
-
+    _face_to_diri = std::make_pair(&fi, bcs[0]);
```

Comment:
Just a few comments here so people who look at this guy know why we are caching!

Reply:
Removed this cache variable in favor of non-mutable map constructed during `initialSetup`